hamocc/mo_vgrid.F90

```diff
@@ -114,6 +115,7 @@ subroutine set_vgrid(kpie,kpje,kpke,pddpo)
 !$OMP END PARALLEL DO
 
 
+  kmle(:,:) =2
```

Comment:
This will overwrite the values set in blom2hamocc every time step.

Reply:
Thanks! That was left over from the first test implementation. It should not be used when importing the kOBL field from BLOM.

Reply:
It might be useful to have the possibility though to set kmle=2 for backwards-compatibility(?). Because the cvmix-routine will not (necessarily) return the index 2 if the old vertical layer structure is activated.

Reply:
I Agree. I can set this in blom2hamocc with a dependency on the value of `vcoord_type_tag`.